<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_atras">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Atras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_adelante">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Adelante</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logos/Unidad2/uat-solo-chico.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logos/Unidad2/descarga.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Edad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_edad"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>196</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pasatiempo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_pasatiempo"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>212</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tipo de sangre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_sangre"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_nombre"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="img_persona">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>91</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/logos/Unidad2/66d822884e7171e9fb9cfadb8daf80b6.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
